src/components/MagicCodeInput.js

```diff
@@ -48,6 +48,12 @@ const propTypes = {
 
     /** Specifies the max length of the input */
     maxLength: PropTypes.number,
+
+    /** Specifies if the keyboard should be disabled */
+    disableKeyboard: PropTypes.bool,
```

Comment:
We prefer `isSomething` or `shouldSomething`, when a value is boolean. Also, `disableKeyboard` sounds like a function to do disable.
```suggestion
    isKeyboardDisable: PropTypes.bool,
```

Reply:
This is literally what `TextInput` component widely used in the app expects. Prop name is 1:1 the same

Reply:
Yeah, I see. I just remember - https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#boolean-variables-and-props. This case could be an exception.

Reply:
k, ill change it